extensions/blocks/tiled-gallery/icons.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+
+/**
+ * @todo: Replace with `@wordpress/icons` when available
```

Comment:
Due to backward compatibility support, we'll need to wait a bit until these are available in WP.